<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkSegmentAnythingGUIControls</class>
 <widget class="QWidget" name="QmitkSegmentAnythingGUIControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkSegmentAnythingToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="basicSettingsLayout">
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="welcomeNote">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to Segment Anything Model (SAM) tool in MITK. [Experimental]&lt;/p&gt;&lt;p&gt;Please note that this is only an interface to SAM. MITK does not ship with SAM. Make sure to have a working internet connection to install SAM via MITK.
		   &lt;/p&gt;&lt;p&gt;Refer to &lt;a href=&quot;https://segment-anything.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://segment-anything.com&lt;/span&gt;&lt;/a&gt; to learn everything about the SAM.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="instructLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>
Press SHIFT+Left-click for positive seeds.
Press SHIFT+Right-click for negative seeds.
Press DEL to remove last seed.
        </string>
       </property>
      </widget>
     </item>

    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="advancedGroupBox" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" stdset="0">
      <string>Preferences</string>
     </property>
     <property name="flat" stdset="0">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight" stdset="0">
      <number>5</number>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="gpuSpinLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>GPU Id:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="ctkComboBox" name="gpuComboBox"/>
        </item>
		
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Reset Picks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="activateButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Activate SAM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="samProgressBar">
     <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">ctkComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleGroupBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
